extensions/shared/external-media/media-browser/media-item.js

```diff
@@ -16,6 +16,15 @@ function MediaItem( props ) {
 		}
 	}, [ props.onClick ] );
 
+	// Catch space and enter keypresses
```

Comment:
If you get to it, [WordPress JS docs standards add a trailing period to inline comments](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#single-line-comments).